x-pack/plugins/dashboard_enhanced/public/services/drilldowns/abstract_dashboard_drilldown/components/dashboard_drilldown_config/dashboard_drilldown_config.tsx

```diff
@@ -6,52 +6,45 @@
  */
 
 import React from 'react';
-import { EuiFormRow, EuiSwitch, EuiComboBox, EuiComboBoxOptionOption } from '@elastic/eui';
+import { EuiFormRow, EuiComboBox, EuiComboBoxOptionOption } from '@elastic/eui';
 import {
-  txtChooseDestinationDashboard,
-  txtUseCurrentFilters,
-  txtUseCurrentDateRange,
-  txtOpenInNewTab,
-} from './i18n';
+  withSuspense,
+  LazyDashboardDrilldownOptionsComponent,
```

Comment:
`LazyDashboardDrilldownOptionsComponent` is not exported. I think you want this:

```suggestion
  DashboardDrilldownOptionsComponent,
```

Reply:
Good catch - that was remaining from when I was doing the lazy importing a bit differently 😅